### DISCUSSION TITLE
Anyone can tell why or help !!! thanks a lot !! trouble : vsgconv missed png files' transparency info !!!

### DISCUSSION THREAD

**Discussion by ytf11** [vsg-dev/VulkanSceneGraph#884] [General]
I have converted (vsgconv) png files with transparency to vsgb or svgt files successfully !!! Unfortunately, transparency info is missed with both extensions !!! Can anyone tell me what's wrong with me ???? 

I typed the following in terminal on mac ( vsgconv was built) ,

vsgconv xxxx.png xxxx.vsgb or,
vsgconv xxxx.png xxxx.vsgt
both files were created and I could try both as textures in vsg examples but transparency was gone or wrong !!!

Considering transparency texture is so needed in game development, I am eager to know what's wrong with me ??? 
Thanks a lot !!!

**Comment by robertosfield** [comment]
Subject lines are best kept on topic, in your case "missing png transparency info" would have been appropriate, all the rest of breathless text you added doesn't help anyone.  Many of the community are professionals so matter of fact presentation is best.

The body of the message doesn't really help us too much either.  All we can guess is some png file doesn't have a transparency setting you were expecting.

The difficulty for us in helping you is that we neither have a png file example that you are having a problem with, nor what you mean be transparency in this context.

A png is just a 2D image format, and the format can be a range of pixel formats, some will have alpha values some not.  alpha values are normally what it used when blending to create a transparency effect. but when rendering if you don't have blending enabled you won't see any transparency effect.

vsgconv doesn't create a full scene graph when it loads an image, it'll simply write out the vsg::Data that represents just the image, this in itself when be renderable.

We left wondering what else you might be doing and why you think that the image somehow isn't being converted with transparency.  Have to ask questions to figure out what you might be doing is far less efficient than you explaining more precisely what you are doing and what you mean.  

Ideally tell us all the steps we can go through to recreate the issue you think exists and we can walk though the same steps and see what may be going wrong.  Providing the "problem" png file would also be very useful.
